Resources/doc/reference/installation.rst

```diff
@@ -25,7 +25,7 @@ Retrieve the bundle with composer:
 
 .. code-block:: bash
 
-    $ php composer.phar require sonata-project/media-bundle
+    $ composer require sonata-project/media-bundle
```

Comment:
> maybe we could do something like this with dev-kit?
> Doing it manually would be a lot of work with nearly no benefit :disappointed:

It would be great, but how?

> I'd say leave it as is, that's not that bad…

It is not a big deal indeed, but the `$` means a prompt. A prompt is not always a `$`.

For easy copy/paste, only the command should be here IMHO.